app/controller.py

```diff
@@ -66,8 +67,10 @@ def changeToSaveAs(self):
     self.host.changeFocusTo(self.host.interactiveSaveAs)
 
   def doCommand(self, ch):
+    #Check the commandSet for the input with both its string and integer representation.
     self.savedCh = ch
-    cmd = self.commandSet.get(ch)
+    keyName = app.curses_util.cursesKeyName(ch)
+    cmd = self.commandSet.get(keyName) or self.commandSet.get(ch)
```

Comment:
Oh yeah I was thinking about this, but I decided to do it the other way because a lot of the commands were strings before. Do you think that I should change back the CTRL + CHAR commands back to an integer value? The reason I ask is because I'm not sure if the ascii value will stay consistent across platforms but the string value seems consistent, at least for control keys.